vehicle/mkvii/software/dashboard/dashboard_config.h

```diff
@@ -0,0 +1,38 @@
+#include "libs/adc/api.h"
+#include "libs/can/api.h"
+#include "libs/gpio/api.h"
+#include "libs/gpio/pin_defs.h"
+#include "libs/timer/api.h"
+#include <avr/interrupt.h>
+#include <avr/io.h>
+
+#define RTD_BUZZ_TIME 400 // hundredths of a second
+#define DASHBOARD_TIME 100
+// Pin definitions - old, def change
+gpio_t IMD_LED = PC7;
+gpio_t BMS_LED = PC6;
+gpio_t HV_LED = PD5;
+
+gpio_t START_BTN = PC6; 
+
+gpio_t RTD_BUZZER_LSD = PC4;
+gpio_t RTD_BUTTON_LED= PC5;
+gpio_t HEARTBEAT_LED= PC1;
+
+gpio_t BOTS_SS= PB2;
+gpio_t ESTOP_SS= PB3;
+
+// Timer setup
+void timer0_callback(void);
+
+timer_cfg_s timer0_cfg = {
```

Comment:
Please add a comment here briefly explaining your timer math and what the frequency is